ccollect

```diff
@@ -242,6 +272,14 @@ while [ "$#" -ge 1 ]; do
     shift
 done
 
+# check that MAX_JOBS is natural number > 0
+# empty string means run all in parallel
+echo "${MAX_JOBS}" | awk '/^$/ { exit 0 } /^[1-9][0-9]*$/ { exit 0 } { exit 1 }'
```

Comment:
if echo ... | grep 'insert-correct-posix-digit-class-here'; then ... is probably easier